### PR TITLE
Add Cloud (TSM) service notices and MQTT EOL notice

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud/reference/service-notices/_index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: InfluxDB Cloud (TSM) service notices
+description: >
+  Important service notices related in to the InfluxDB Cloud (TSM) platform.
```
Comment (jstirnaman) [inline content/influxdb/cloud/reference/service-notices/_index.md:4]:
```suggestion
  Important service notices related to the InfluxDB Cloud (TSM) platform.
```

File: content/influxdb/cloud/reference/service-notices/_index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+menu:
+  influxdb_cloud_ref:
+    name: Service notices
+weight: 12
+---
+
+Important service notices related in to the InfluxDB Cloud (TSM) platform.
```
Comment (jstirnaman) [inline content/influxdb/cloud/reference/service-notices/_index.md:11]:
```suggestion
Important service notices related to the InfluxDB Cloud (TSM) platform.
```

File: content/influxdb/cloud/reference/service-notices/native-collector-mqtt-eol.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+It was only available to a limited number of existing customers who were
+actively using it.
+
+### Why is this feature being EOL'd?
+
+The adoption of this feature was lower than anticipated given most customers
+continued to use [Telegraf](/telegraf/v1/), InfluxData’s data collection agent,
+which provides more features for MQTT ingestion than this feature did.
+After a thorough assessment, we determined that our customers can have a similar
+(and even superior) experience with the
```
Comment (jstirnaman) [inline content/influxdb/cloud/reference/service-notices/native-collector-mqtt-eol.md:44]:
```suggestion
(and even superior) experience using the
```

File: content/influxdb/cloud/reference/service-notices/native-collector-mqtt-eol.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+[MQTT Consumer Telegraf input plugin](https://www.influxdata.com/integration/mqtt-telegraf-consumer/).
+
+### What alternatives are available in light of this EOL announcement?
+
+We recommend using the [MQTT Consumer Telegraf input plugin](https://www.influxdata.com/integration/mqtt-telegraf-consumer/)
+as an alternative to the Native Collector - MQTT feature.
+InfluxData’s very popular open source data collection agent with 300+ plugins,
+including MQTT.
```
Comment (jstirnaman) [inline content/influxdb/cloud/reference/service-notices/native-collector-mqtt-eol.md:52]:
```suggestion
We recommend using [Telegraf](/telegraf/v1/) with the [MQTT Consumer Telegraf input plugin](https://www.influxdata.com/integration/mqtt-telegraf-consumer/)
as an alternative to the Native Collector - MQTT feature.

Telegraf is InfluxData’s popular open source data collection agent that comes with 300+ plugins for data input and output,
including MQTT.
```

File: content/influxdb/cloud/reference/service-notices/native-collector-mqtt-eol.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+### What are the expected next steps?
+
+The Native Collector - MQTT feature will continue to work and function until
+**April 30th, 2024**.  We will not be doing any new product development or
+enhancements to the feature in the meantime. After the EOL date, the feature
+will be decommissioned and any MQTT subscriptions that you have made will no
+longer receive and ingest data into InfluxDB.
```
Comment (jstirnaman) [inline content/influxdb/cloud/reference/service-notices/native-collector-mqtt-eol.md:60]:
```suggestion
The Native Collector - MQTT feature will continue to work and function until
**April 30th, 2024**.
In the meantime, we won't release any new enhancements to the feature.
After the EOL date, we will decommission the feature, and any MQTT subscriptions that you have will no
longer receive and ingest data into InfluxDB.
```

**Review by jstirnaman — CHANGES_REQUESTED** [review]
Some typo and grammar fixes.

**Comment by sanderson** [comment]
@garylfowler Let me know when you're ready for this content to be available on the public docs.

**Review by garylfowler — APPROVED** [review]

**Pull request merged**